libtcmu.c

```diff
@@ -381,6 +381,10 @@ void tcmu_dev_flush_ring(struct tcmu_device *dev)
 	tcmu_dev_dbg(dev, "ring clear\n");
 }
 
+bool tcmu_dev_oooc_supported(struct tcmu_device* dev) {
```

Comment:
Just following the existing coding style and put the opening "{" on a new line when declaring a function here. Check out tcmu_dev_get_private in the same file for an example. Thanks.

Reply:
Thanks for taking a look Mike. I've updated the PR to fix the code style.